StripePaymentsUI/StripePaymentsUI/Source/Internal/Categories/STPCardBrand+PaymentsUI.swift

```diff
@@ -26,17 +27,21 @@ extension STPCardBrand {
         return NSAttributedString(attachment: brandImageAttachment)
     }
 
-    func cardBrandItem(theme: ElementsAppearance = .default, maxWidth: CGFloat? = nil) -> DropdownFieldElement.DropdownItem {
+    func cardBrandItem(theme: ElementsAppearance = .default, isAllowed: Bool, maxWidth: CGFloat? = nil) -> DropdownFieldElement.DropdownItem {
```

Comment:
Would be good to keep a consistent nomenclature  of `disallowed` instead of introducing `isAllowed`

Reply:
Yeah agree!